### DISCUSSION TITLE
CefWing创建时机以及作用

### DISCUSSION THREAD

**Discussion by L-Super** [CefView/QCefView#242] [Q&A] [answered]
想问一下，程序运行后，会同时有多个CefWing进程，他们是何时创建的呢？CEF程序是有多个同名进程，而我不太理解这里为什么是单独的CefWing进程，它的作用是什么？

**Comment by sdcarterchen** [comment, marked as answer]
这个是在CefInitialize时设置的，对应的是cefsettings是browser_subprocess_path。CefWing是多进程模式创建子进程用的，根据不同的type创建不同的进程。
